packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/PaymentMethods/Payments/index.tsx

```diff
@@ -97,7 +97,13 @@ class Payments extends PureComponent<InjectedFormProps<{}, Props> & Props> {
           />
         )
       case 'FUNDS':
-        return <></>
+        return (
+          <Icon
+            size='32px'
+            color='fiat'
+            name={value.currency === 'EUR' ? 'eur' : 'gbp'}
```

Comment:
value.currency.toLowerCase()

Reply:
nope that doesn't work due to typing...

Reply:
value.currency.toLowerCase() as key of DefaultTheme

Reply:
we had to make it something like `name={value.currency.toLowerCase() as 'eur' | 'gbp'}`